### PR TITLE
[Polygon Edge] Mark flag as deprecated

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

File: docs/edge/get-started/cli-commands.md [hunk 1]
```diff
@@ -541,6 +541,9 @@ Sets block production time in seconds. Default: `2`
   </TabItem>
 </Tabs>
 
+:::caution This flag is deprecated for any Polygon Edge version ≥ v0.5.0.
```
Comment (zivkovicmilos) [inline docs/edge/get-started/cli-commands.md:544]: You can delete it completely from the docs, no need to note that it's deprecated

Comment (Aleksao998) [reply docs/edge/get-started/cli-commands.md:544]: Agree

Comment (laviniat1996) [reply docs/edge/get-started/cli-commands.md]: Deleted 459918b090e389dfa80d07b670404c4e08103e9d

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

**Pull request merged**